packages/build/src/download-center.ts

```diff
@@ -67,11 +67,17 @@ const CONFIG = `
           "download_link": "https://downloads.mongodb.com/compass/mongosh-{{version}}-linux.tgz"
         },
         {
-          "arch": "x64",
+          "arch": "amd64",
           "os": "debian",
           "name": "Debian 64-bit",
           "download_link": "https://downloads.mongodb.com/compass/mongosh_{{version}}_amd64.deb"
         }
+        {
+          "arch": "amd64",
```

Comment:
I am currently building this for `amd64` as well. If we want to be building for `x86-64` instead, let me know. It's easily changeable.

Reply:
yep, also don't mind this.